### ISSUE TITLE
Stuck at blank screen

### ISSUE THREAD

**Issue by Lunderberg** [dnschneid/crouton#1291]
Starting this morning, on an HP Chromebook 14, when running `startgnome` from the crosh shell, the loading process gets stuck at a blank screen.  This persists after having updated the chroot to the latest version of crouton, according to [these instructions](https://github.com/dnschneid/crouton#a-new-version-of-crouton-came-out-my-chroot-is-therefore-obsolete-and-sad).  Previously, the black screen would appear for a few seconds, followed by the desktop of the chroot appearing.

There were no changes to the chroot environment made since the last working version.  Looking at the [Chrome OS changelog](http://googlechromereleases.blogspot.com/), there have been no new updates within the past week that would have been applied.

When the blank screen appears, I can use Ctrl-Alt-F1 to return to ChromeOS.  I can use Ctrl-Alt-F2 followed by Ctrl-Alt-F3, the key combinations that would previously return to the chroot, to return to the blank screen.  The following are the messages shown in crosh when I return.  Unfortunately, I do not have a copy of the messages shown from a working copy.

```
Initializing built-in extension XKEYBOARD
Initializing built-in extension XC-MISC
Initializing built-in extension SECURITY
Initializing built-in extension XINERAMA
Initializing built-in extension XFIXES
Initializing built-in extension RENDER
Initializing built-in extension RANDR
Initializing built-in extension COMPOSITE
Initializing built-in extension DAMAGE
Initializing built-in extension MIT-SCREEN-SAVER
Initializing built-in extension DOUBLE-BUFFER
Initializing built-in extension RECORD
Initializing built-in extension DPMS
Initializing built-in extension Present
Initializing built-in extension DRI3
Initializing built-in extension X-Resource
Initializing built-in extension XVideo
Initializing built-in extension XVideo-MotionCompensation
Initializing built-in extension SELinux
Initializing built-in extension XFree86-VidModeExtension
Initializing built-in extension XFree86-DGA
Initializing built-in extension XFree86-DRI
Initializing built-in extension DRI2
Loading extension GLX
Timeout waiting for extension to connect.
croutonwebsocket error: Error: not connected.Error: target STRING not available
croutonwebsocket error: EError: not connected.
```

If I use `startcli` instead of `startgnome`, I can access the chroot with the command line, and have observed no strange behaviors there.  The output from `croutonversion` when run from the command line is as follows.

```
crouton: version 20141231011135~master:992afe3e
release: trusty
architecture: amd64
targets: gnome,core,extension,cli-extra,audio,gtk-extra,keyboard
host: version 6310.68.0 (Official Build) stable-channel falco
```

**Comment by dnschneid** [comment]
Do you still hit this with the latest crouton/chromium OS?

**Comment by Lunderberg** [comment]
Unfortunately, I no longer have a copy of the particular chroot that exhibited this issue.  Due to limited hard drive space, when I tested making a new chroot with the same settings and the same version of crouton as before, I needed to first delete the previous one.  The new chroot ran without error, and has been working since.

I do apologize for not keeping a copy of the defective chroot, as I would have liked to have been able to reproduce the issue and help find the original cause.
